source/Xunit.Gherkin.Quick.ProjectConsumer/Addition/SearchUsers.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Gherkin.Ast;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Xunit.Abstractions;
+
+namespace Xunit.Gherkin.Quick.ProjectConsumer.Addition
```

Comment:
Can you keep it under different folder? e.g. Users. Along with the UserModel and feature file too, in it please.

Reply:
And also change the namespace accordingly.